.github/workflows/package.yml

```diff
@@ -58,20 +62,35 @@ jobs:
           git submodule foreach --recursive git clean -ffdx
           git submodule foreach --recursive git reset --hard
 
-      - name: check bash
-        run: yarn run check:bash:ci
-
       - uses: actions/setup-node@v4
         with:
           node-version: ${{ matrix.node }}
-          architecture: ${{ matrix.host }}
```

Comment:
With the achitecture/host removed, what changes?

Reply:
As per GitHub [documentation](https://github.com/actions/setup-node), `setup-node` uses system architecture by default, so this option does nothing.